src/menu/dto/menu.dto.ts

```diff
@@ -8,19 +8,19 @@ import {
   ValidateNested,
 } from 'class-validator';
 
-export class Modification {
+export class ModificationGroup {
```

Comment:
Let's rename this. Otherwise, it will conflict with prisma type. let's call this file `create-menu-item.dto.ts`

And rename it to `CreateMenuItemDto`. And rename `ModificationGroup` to `ModificationGroupDto`